web-gui/src/app/components/courses/detail-course/update-course-dialog/update-course-dialog.component.ts

```diff
@@ -123,14 +94,23 @@ export class UpdateCourseDialogComponent implements OnInit {
    * @param standardTaskType Course standartTaskType
    * @param userDateAllowed Course userDataAllowed
    */
-  checkRequiredFields(couseID: number, name: string, description: string, standardTaskType:string, userDateAllowed: boolean):boolean{
-    return true
-    if (couseID !== null && couseID !== undefined)
-      if (name !== undefined && name != null && name.trim() !== "" && name.length <= this.courseNameMaxLength && name.length >= this.coursenameMinLength)
-        if (standardTaskType !== undefined && standardTaskType !== null && standardTaskType.trim() !== "")
-          if (userDateAllowed !== undefined && userDateAllowed !== null)
-            if (description.length <= this.courseDescriptionMaxLength)
-              return true
-    return false
+  checkRequiredFields(couseID: number, name: string, description: string, standardTaskType: string, userDateAllowed: boolean): boolean {
```

Comment:
Do we still need this method? I have thought that by using form validators we can avoid using this method.